index.js

```diff
@@ -376,11 +378,9 @@ app.post('/api/door', Validator.body(['coreid', 'data']).exists(), async (reques
           doorSignal = DOOR_STATUS.OPEN
         } else if (signal === IM21_DOOR_STATUS.CLOSED || signal === IM21_DOOR_STATUS.HEARTBEAT_CLOSED) {
           doorSignal = DOOR_STATUS.CLOSED
-        } else if (signal === IM21_DOOR_STATUS.LOW_BATT && !lowBatteryAlertTimeout(location.sentLowBatteryAlertAt)) {
+        } else if (signal === IM21_DOOR_STATUS.LOW_BATT) {
```

Comment:
I think that I changed the look of this function quite a bit to fix https://app.clickup.com/t/4avrue
It would might be worth it to rebase off devenv to make sure that you're working on the latest.